theme/static/style/style.css

```diff
@@ -430,6 +430,21 @@ pre code {
     float: right;
 }
 
+.openexpo-img {
+    width: 200px;
+    height: 200px;
+    background-image: url('/images/conf/openexpo.png');
+    background-repeat: no-repeat;
+    display: inline-block;
+    float: left;
+    margin-left: 1.5rem;
+}
```

Comment:
With the exception of `background-image` this is the same as `fosdem-img`. Use a generic class definition for the layout + a specific one for the background image and don't repeat.